features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -1313,6 +1313,12 @@
   "console.approvals.scopes.read": ["internal_humantask_view"],
   "console.approvals.scopes.update": ["internal_humantask_view"],
   "console.approvals.scopes.delete": ["internal_humantask_view"],
+  "console.workflows.enabled": false,
+  "console.workflows.scopes.feature": ["console:workflowApprovals"],
+  "console.workflows.scopes.create": ["internal_humantask_view"],
+  "console.workflows.scopes.read": ["internal_humantask_view"],
+  "console.workflows.scopes.update": ["internal_humantask_view"],
+  "console.workflows.scopes.delete": ["internal_humantask_view"],
```

Comment:
Is using the same scope for all operations intentional?

Reply:
This should be updated with new APIs.
 
internal_workflow_create, internal_workflow_view etc..

We can remove the entries related approvals.
We won't need to protect the approval API by scopes. It is a me API which provided to view, update the approvals assigned / eligible for the authenticated user.

"console.approvals.scopes.delete": ["internal_humantask_view"],